### DISCUSSION TITLE
Numpy 2.0.0 support?

### DISCUSSION THREAD

**Discussion by bzm3r** [ibis-project/ibis#9485] [Q&A] [answered]
Currently, `ibis-framework` (whether or `<= 9.1.0` or `<= 10.0.0.dev120`) relies on `1.23.2 <= numpy < 2`. 

Are there any plans to support `numpy == 2`?

**Comment by gforsyth** [comment, marked as answer]
Hey @bzm3r !

We just added support in https://github.com/ibis-project/ibis/pull/9395

**Reply by bzm3r** [reply]
@gforsyth Nice! How would I go about using the version of Ibis with numpy 2.0 support? Is it one of the newer dev releases?

**Reply by gforsyth** [reply]
You can wait for the 9.2 release, or you can try out a pre-release by installing using `pip install -U --pre ibis-framework`

**Reply by judahrand** [reply]
Is there an expected release cadence? When should we expect new versions to be released?

**Reply by lostmygithubaccount** [reply]
we're trying to release every few weeks, personally bumping the numpy version alone seems worth releasing 9.2.0 soon? though most of the team is at the SciPy conference this week, so I'm not sure if we can do it this week
